### PR TITLE
Bug: Fix bug for incorrect datatype judgement in cublas-function source code

### PR DESCRIPTION
**Pull request by yukirora**
**Description**
Fix bug for incorrect datatype judgement in cublas-function source code.

### CODE REVIEW

**Comment by rhewett** [comment]
This looks good to me.

**Review by cp5555 — APPROVED** [review]

File: superbench/benchmarks/micro_benchmarks/cublas_function/cublas_function_helper.h [hunk 1]
```diff
@@ -175,13 +175,25 @@ void from_json(const json &j, CublasFunction &fn) {
...
+        } else {
+            auto datatype = j.at("datatype").get<std::string>();
+            fn.set_datatype(datatype);
+        }
+
+    } catch (std::exception &e) {
+        throw std::runtime_error("datatype is required for cublasGemmEx and cublasGemmStridedBatchedEx");
+    }
+    try {
+        if (str.find("use_tensor_core") == std::string::npos) {
```
Comment (rhewett) [inline superbench/benchmarks/micro_benchmarks/cublas_function/cublas_function_helper.h:189]: I assume this catches the case where the required value is missing?

I think that, if it is required, it should fail if it is missing.

**Comment by rhewett** [comment]
I'm going to approve this as-is.  

I do prefer failing if a required parameter is missing rather than silently going to a default, but I will not block it on that.

**Review by rhewett — APPROVED** [review]

File: superbench/benchmarks/micro_benchmarks/cublas_function/cublas_function_helper.h (hunk 1 above)
Comment (yukirora) [reply superbench/benchmarks/micro_benchmarks/cublas_function/cublas_function_helper.h:189]: some functions like pure sgemm do not need this parameter, so this parameter is optional not required in some cases

**Pull request merged**